apps/speed.c

```diff
@@ -54,68 +57,23 @@
 #ifndef OPENSSL_NO_DEPRECATED_3_0
 #include <openssl/aes.h>
 #endif
-#ifndef OPENSSL_NO_CAMELLIA
-# include <openssl/camellia.h>
-#endif
 #ifndef OPENSSL_NO_MD2
```

Comment:
Why is this still here?  Later changes have removed the OPENSSL_NO_MD2 guards.

Reply:
It's for the MD2_options() call. Similarly for others.

Reply:
Okay, makes sense.

Reply:
We should have an EVP  way of getting that information so these calls can all go - this should be a pure EVP-only application now.

Reply:
Wouldn't these be considered an implementation detail that can stay hidden in the provider?

Reply:
Either this should be a separate issue, or we should simply remove those _options() calls. Also we still include some of the algorithm-specific headers for stuff like some constants which are not even deprecated.

Reply:
Some of the constants were intentionally not deprecated.  Things like e.g. `MD2_DIGEST_LENGTH` were retained because there is no chance they'll ever change and thus have a support overhead of near zero.

Reply:
I think we should remove the options calls. They're not useful.

Reply:
That works for me. I'll wait till Monday to see if anybody objects.

Reply:
_options calls removed (except for BN_options which is not deprecated)